framework/graphics/dx12_util.cpp

```diff
@@ -1023,7 +1024,6 @@ uint64_t GetAvailableCpuMemory(double max_usage)
         GFXRECON_LOG_ERROR("Failed to get available virtual memory");
     }
 
-    // Only limit by available physical memory if max_usage <= 1.0.
     uint64_t avail_phys = std::numeric_limits<uint64_t>::max();
```

Comment:
This comment `// Only limit by available physical memory if max_usage <= 1.0.` was removed but it appears to still be true.